JustSaying/Microsoft/Extensions/DependencyInjection/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,200 @@
+using System;
+using System.ComponentModel;
+using JustSaying;
+using JustSaying.AwsTools;
+using JustSaying.AwsTools.QueueCreation;
+using JustSaying.Fluent;
+using JustSaying.Messaging.MessageHandling;
+using JustSaying.Messaging.MessageSerialization;
+using JustSaying.Messaging.Monitoring;
+using JustSaying.Models;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.Extensions.DependencyInjection
```

Comment:
Could we add a `JustSaying.Extensions.DependencyInjection.csproj` project as part of this? (and your thoughts on naming)

Reply:
Plan was to do that separately once/if this gets merged, then JustSaying itself doesn’t get this dependency and remains “clean”.